docs/developer-docs/latest/development/backend-customization/models.md

```diff
@@ -494,6 +495,34 @@ The `tableName` key defines the name of the join table. It has to be specified o
 
 :::::
 
+#### Custom fields
+
+[Custom fields](/developer-docs/latest/development/custom-field.md) extend Strapi’s capabilities by adding new types of fields to content-types. Custom fields are explicitly defined in the [attributes](#model-attributes) of a model with `type: customField`.
+Custom fields attributes also accept:
+
+- the `customField` additional parameter to mention the registered custom field to use, using:
+  - either the `plugin::plugin-name.fieldname` format for a custom field created by a plugin
+  - or the `global::api-name.fieldname` for a custom field specific to the current Strapi application
```

Comment:
Technically if a custom field is created by a local, unpublished plugin, it would also be specific to the application. But phrasing it like you did makes it more understandable so it's not an issue, I just wanted to point it out.

More importantly, the UID is just `global::fieldname`, since a custom field is not tied to a specific API

Reply:
Thanks, Rémi!
Do you think it would lift any ambiguity if I reword it this way:

> the `customField` additional parameter […], using:
>   - either the `plugin::plugin-name.fieldname` format for a custom field created by a plugin installed from the Marketplace
>   - or the `global::fieldname` for a custom field specific to the current Strapi application (which includes custom fields created by local, unpublished plugin)

Reply:
I think it's the other way around actually, sorry for the misunderstanding: a local unpublished plugin would still get the plugin:: prefix

Reply:
Ah, I see 😅 Thanks!
So I think I'll keep the initial wording and just fix the uid format for a "global" custom field.